IKOR-GmbH/sip-framework
# Packaging markdown files into jar. Structure for markdown files within adapter is changed.

# Description

Added markdown files in jar. When running the adapter with jar, markdown files from adapter application module should be included and shown under /actuator/info endpoint.

## Type of change


# How Has This Been Tested?

Testing steps:
1. Create new adapter from archetype.
2. do mvn clean install in your adapter
3. Run the adapter and check if everything is there under /actuator/info
4. Run the jar of adapter and check actuator/info


# Checklist:
